.gitignore

```diff
@@ -9,6 +9,7 @@ lerna-debug.log
 node_modules
 .vscode/*
 package-lock.json
+boilerplate/yarn.lock
```

Comment:
The reason we added this to the CLI's .gitignore instead of the boilerplate's .gitignore is that we _only_ want to ignore it as a boilerplate, not when a new app is generated.